webpack.config.js

```
@@ -8,67 +7,50 @@ module.exports = {
   ],
   resolve: {
     extensions: ['.js', '.vue', '.json'],
     alias: {
       'vue$': 'vue/dist/vue.esm-bundler.js',
-      'three/examples/jsm/controls/OrbitControls': 'three/addons/controls/OrbitControls.js',
-      'three/examples/jsm/libs/stats.module': 'three/addons/libs/stats.module.js',
+      // Fix three.js module resolution for packages that import without .js extension
+      'three/examples/jsm/controls/OrbitControls$': 'three/examples/jsm/controls/OrbitControls.js',
+      'three/examples/jsm/libs/stats.module$': 'three/examples/jsm/libs/stats.module.js',
       // Provide empty modules for Node.js built-ins
       'fs': path.resolve(__dirname, 'src/polyfills/empty.js'),
```

ESLint

[error] '__dirname' is not defined.

(no-undef)